gridmap/job.py

```diff
@@ -828,7 +833,7 @@ def _append_job_to_session(session, job, temp_dir='/scratch/', quiet=True):
 
 
 def process_jobs(jobs, temp_dir='/scratch/', white_list=None, quiet=True,
-                 max_processes=1, local=False):
+                 max_processes=1, local=False, force_cluster=False):
```

Comment:
It's pretty minor, but I'd greatly prefer the name `require_cluster` over `force_cluster`.